WAR for issue https://github.com/horovod/horovod/issues/2842
diff --git a/horovod/keras/__init__.py b/horovod/keras/__init__.py
index f04c6eb..2957e64 100644
--- a/horovod/keras/__init__.py
+++ b/horovod/keras/__init__.py
@@ -119,7 +119,7 @@ def broadcast_global_variables(root_rank):
     return _impl.broadcast_global_variables(K, root_rank)
 
 
-def allreduce(value, name=None, average=True, prescale_factor=1.0, postscale_factor=1.0):
+def allreduce(value, name=None, average=True, prescale_factor=1.0, postscale_factor=1.0, op=None, compression=Compression.none):
     """
     Perform an allreduce on a tensor-compatible value.
 
@@ -132,7 +132,7 @@ def allreduce(value, name=None, average=True, prescale_factor=1.0, postscale_fac
         prescale_factor: Multiplicative factor to scale tensor before allreduce.
         postscale_factor: Multiplicative factor to scale tensor after allreduce.
     """
-    return _impl.allreduce(K, value, name, average, prescale_factor, postscale_factor)
+    return _impl.allreduce(K, value, name, average, prescale_factor, postscale_factor, op, compression)
 
 
 def allgather(value, name=None):
